modules/persistent-storage-hostpath-static-provisioning.adoc

```diff
@@ -0,0 +1,64 @@
+// Module included in the following assemblies:
+//
+// * storage/persistent-storage/persistent-storage-vsphere.adoc
+
+[id="vsphere-static-provisioning_{context}"]
+= Statically provisioning hostPath volumes
+
+A Pod that uses a hostPath volume must be referenced by manual (static) provisioning.
+
+.Procedure
+
+. Define the persistent volume (PV). Create a file, `pv.yaml`, with the PersistentVolume object definition:
++
+[source,yaml]
+----
+  apiVersion: v1
+  kind: PersistentVolume
+  metadata:
+    name: task-pv-volume <1>
+    labels:
+      type: local
+  spec:
+    storageClassName: manual <2>
+    capacity:
+      storage: 5Gi
+    accessModes:
+      - ReadWriteOnce <3>
+    persistentVolumeReclaimPolicy: Retain
+    hostPath:
+      path: "/mnt/data" <4>
+----
+<1> The name of the volume. This name is how it is identified by PersistentVolumeClaims or Pods.
+<2> Used to bind PersistentVolumeClaim requests to this PersistentVolume.
+<3> The volume can be mounted as `read-write` by a single node.
+<4> The configuration file specifies that the volume is at `/mnt/data` on the cluster’s node.
+
+. Create the PV from the file:
++
+----
+$ oc create -f pv.yaml
+----
+
+. Define the persistent volume claim (PVC). Create a file, `pvc.yaml`, with the PersistentVolumeClaim object definition:
++
+[source,yaml]
+----
+apiVersion: v1
+kind: PersistentVolumeClaim
+metadata:
+  name: task-pvc-volume
+spec:
+  accessModes:
+    - ReadWriteOnce
+  resources:
+    requests:
+      storage: 1Gi
+  storageClassName: slow
```

Comment:
The PVC's storageClassName mismatch with PV's storageClassName.

Reply:
Updated so they are both using `manual` now.